debian/changelog

```diff
@@ -26,8 +29,14 @@ cloud-init (24.1.3-0ubuntu1~20.04.4) UNRELEASED; urgency=medium
     - d/p/status-retain-recoverable-error-exit-code.patch
     - d/p/revert-551f560d-cloud-config-after-snap-seeding.patch
   * Upstream snapshot based on upstream/main at db859451.
+  * refresh patches:
+    - d/p/cli-retain-file-argument-as-main-cmd-arg.patch
+    - d/p/drop-unsupported-systemd-condition-environment.patch
```

Comment:
these three are duplicates

Reply:
Updated. Also used your name/description for the patch.